python/rgb_hex.py

```diff
@@ -0,0 +1,3 @@
+rgb = lambda r, g, b: "{:02X}{:02X}{:02X}".format(
+    max(0, min(255, r)), max(0, min(255, g)), max(0, min(255, b))
+)
```

Comment:
This is not an acceptable practice in Python.  It is fine in JavaScript.  Nameless functions should only be used in cases where they will not be called by name.  So they should never be stored in a variable, since this gives them a name. 

Good Job learning something new though, and it's good to understand the incorrect ways also, because you'll see plenty of them in the wild.